diff --git a/chrome/browser/ui/views/profiles/profile_menu_view.cc b/chrome/browser/ui/views/profiles/profile_menu_view.cc
index 631efe1e3e57b6789742b6eba8bc6656bc1b55b2..49f2439a2c11d23475c33f2ffa0d9e7948bc547e 100644
--- a/chrome/browser/ui/views/profiles/profile_menu_view.cc
+++ b/chrome/browser/ui/views/profiles/profile_menu_view.cc
@@ -609,3 +609,5 @@ void ProfileMenuView::BuildProfileManagementFeatureButtons() {
                             base::Unretained(this)));
   }
 }
+
+        BRAVE_ADDOPTIONSVIEW_ADD_TOR_EXIT_BUTTON_
\ No newline at end of file
