diff --git a/chrome/browser/ui/views/profiles/incognito_menu_view.cc b/chrome/browser/ui/views/profiles/incognito_menu_view.cc
index e3166ed8eb3e34dc580cc2476ca7c11b20d131a2..0d748d5654404fa8ffcebeeb29a7ab92b15838d9 100644
--- a/chrome/browser/ui/views/profiles/incognito_menu_view.cc
+++ b/chrome/browser/ui/views/profiles/incognito_menu_view.cc
@@ -47,6 +47,7 @@ void IncognitoMenuView::BuildMenu() {
   const SkColor icon_color = provider->GetTypographyProvider().GetColor(
       *this, views::style::CONTEXT_LABEL, views::style::STYLE_PRIMARY);
 
+    BRAVE_BUILDMENU_ADD_TOR_BUTTON
   SetIdentityInfo(
       ColoredImageForMenu(kIncognitoProfileIcon, icon_color),
       /*badge=*/gfx::ImageSkia(),
