diff --git a/chrome/browser/ui/views/profiles/profile_menu_view.h b/chrome/browser/ui/views/profiles/profile_menu_view.h
index c64faf29bb5a310ce8ed1b9c7ad19df522f34563..b78aca0188675a830faba93cfb1887a16a303f10 100644
--- a/chrome/browser/ui/views/profiles/profile_menu_view.h
+++ b/chrome/browser/ui/views/profiles/profile_menu_view.h
@@ -38,6 +38,7 @@ class ProfileMenuView : public ProfileMenuViewBase {
   void BuildMenu() override;
 
  private:
+  BRAVE_PROFILE_MENU_VIEW_H
   friend class ProfileMenuViewExtensionsTest;
 
   // views::BubbleDialogDelegateView:
@@ -50,6 +51,7 @@ class ProfileMenuView : public ProfileMenuViewBase {
   void OnAddressesButtonClicked();
   void OnGuestProfileButtonClicked();
   void OnManageProfilesButtonClicked();
+  virtual
   void OnExitProfileButtonClicked();
   void OnSyncSettingsButtonClicked();
   void OnSyncErrorButtonClicked(sync_ui_util::AvatarSyncErrorType error);
