diff --git a/chrome/browser/importer/external_process_importer_client.cc b/chrome/browser/importer/external_process_importer_client.cc
index 5c56fb75075438213e2ec261ffe4c1833a5c8d16..1e4d774a3058688e3bda6ca6b61296c0cc816383 100644
--- a/chrome/browser/importer/external_process_importer_client.cc
+++ b/chrome/browser/importer/external_process_importer_client.cc
@@ -68,6 +68,7 @@ void ExternalProcessImporterClient::Start() {
   localized_strings.try_emplace(
       IDS_BOOKMARK_BAR_FOLDER_NAME,
       l10n_util::GetStringUTF8(IDS_BOOKMARK_BAR_FOLDER_NAME));
+  BRAVE_START
 
   // If the utility process hasn't started yet the message will queue until it
   // does.
