diff --git a/chrome/browser/themes/theme_service_factory.cc b/chrome/browser/themes/theme_service_factory.cc
index 5a0d5e736cb9fa72d9d764327aa0d549cfed563b..6c306b2e67d4779c0981078a6791f245e629034f 100644
--- a/chrome/browser/themes/theme_service_factory.cc
+++ b/chrome/browser/themes/theme_service_factory.cc
@@ -83,7 +83,7 @@ KeyedService* ThemeServiceFactory::BuildServiceInstanceFor(
 #endif
 
   auto provider = std::make_unique<ThemeService>(static_cast<Profile*>(profile),
-                                                 GetThemeHelper());
+      BRAVE_THEMESERVICEFACTORY_BUILDSERVICEINSTANCEFOR);
   provider->Init();
   return provider.release();
 }
