diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 0cbaf53ecec1d35ad8b0425ad6d44665a9d68d19..780c9b323cc2b4d2b93fc8bfe9bc6597e63a8396 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -760,6 +760,7 @@ if (!is_android) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
