diff --git a/third_party/blink/renderer/core/BUILD.gn b/third_party/blink/renderer/core/BUILD.gn
index 1e7e3825f22b3964e2056d267d08f68b91e0fad9..784ea921e8f5732958731d5087eec88a2f51d308 100644
--- a/third_party/blink/renderer/core/BUILD.gn
+++ b/third_party/blink/renderer/core/BUILD.gn
@@ -214,6 +214,7 @@ component("core") {
     "//url",
     "//v8",
   ]
+  public_deps += [ "//brave/third_party/blink/renderer" ]
   deps = [
     "//third_party/blink/public/common",
     "//third_party/blink/renderer/platform",
